packages/block-library/src/index.native.js

```diff
@@ -160,7 +166,7 @@ export const registerCoreBlocks = () => {
 		latestPosts,
 		devOnly( verse ),
 		cover,
-		pullquote,
+		iOSOnly( pullquote ),
```

Comment:
We still want the pullquote block showing up on dev builds. This is entirely disabling it on Android though, right?